Framework/DataHandling/src/LoadMask.cpp

```diff
@@ -246,17 +464,17 @@ void LoadMask::componentToDetectors(
 
     g_log.debug() << "Number of Children = " << children.size() << '\n';
 
-    int32_t numdets = 0;
-    int32_t id_min = 1000000;
-    int32_t id_max = 0;
+    size_t numdets(0);
+    detid_t id_min = std::numeric_limits<Mantid::detid_t>::max();
+    detid_t id_max = 0;
```

Comment:
I wonder why mix the `numdets(0)` type initialization with the assignment style initialization. Shouldn't one choose a single style and stick to it?